icons/src/fullcolor/render-bitmaps-verbose.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/python3
+# coding: utf-8
+#
+# Legal Stuff:
+#
+# This file is part of the Moka Icon Theme and is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free Software
+# Foundation; version 3.
+#
+# This file is part of the Moka Icon Theme and is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, see <https://www.gnu.org/licenses/lgpl-3.0.txt>
+#
+#
+# Thanks to the GNOME icon developers for the original version of this script
+import os
+import sys
+import xml.sax
+import subprocess
+import argparse
+
+INKSCAPE_PATH = '/usr/bin/inkscape'
+OPTIPNG_PATH = '/usr/bin/optipng'
```

Comment:
Since we are reworking this, I would suggest to get rid of the full path and just look for the binary wherever it is installed in the system. This way we should also be able to use snap/flatpak install.